test/__fixtures__/infer-plugins-ext-dir-with-overrides-and-default-options/.prettierrc.mjs

```diff
@@ -0,0 +1,12 @@
+export default {
+  overrides: [
+    {
+      // TODO (43081j): in prettier, this was `*.foo` and would successfully
+      // match `src/*.foo`. In prettier CLI, this is not the case
+      files: ["**/*.foo"],
```

Comment:
@fabiospampinato this one seems like something we should discuss/decide before we merge these tests

it seems prettier treats `*.foo` as if it was `**/*.foo`, whereas we treat it as `./*.foo`

i think we're correct, but this means a whole bunch of prettier configs may fail with our CLI. maybe we should do a special case that transforms `"*.ext"` into `"**/*.ext"`?

Reply:
Pretty much, just the paths changed to the mock/fixture extensions. The rest should be the same